javascript/javascript3/week1/apiresponse.js

```diff
@@ -0,0 +1,46 @@
+/* 
+2. Postal PIN code API http://www.postalpincode.in/Api-Details
+
+Postal PIN Code API responds the details of post offices by searching a specific postal pin code
+or a branch name within India. The responded json data shows the number of post offices, the type of post office
+whether it is "Main/Head" or a "Sub" post office and it's complete address means which city and state that
+post office belongs to. For example:
+
+a. GET https://api.postalpincode.in/pincode/{PINCODE}  ---- Which gives the number of postal offices and it's
+ addresses and related sub post offices based on the given pincode. The api responded json data is in array and
+ the data structure is "array of objects". Below is the json response data.
+
+
+[{"Message":"Number of pincode(s) found:3","Status":"Success","PostOffice":[{"Name":"Chaithanyapuri Colony",
+"Description":null,"BranchType":"Sub Post Office","DeliveryStatus":"Non-Delivery","Circle":"Andhra Pradesh",
+"District":"K.V.Rangareddy","Division":"Hyderabad South East","Region":"Hyderabad City","Block":"Saroornagar",
+"State":"Telangana","Country":"India","Pincode":"500060"},{"Name":"Dilsukhnagar Colony","Description":null,
+"BranchType":"Sub Post Office","DeliveryStatus":"Non-Delivery","Circle":"Andhra Pradesh","District":"K.V.Rangareddy",
+"Division":"Hyderabad South East","Region":"Hyderabad City","Block":"Saroornagar","State":"Telangana",
+"Country":"India","Pincode":"500060"},{"Name":"P & T Colony (K.V.Rangareddy)","Description":null,"BranchType":
+"Sub Post Office","DeliveryStatus":"Delivery","Circle":"Andhra Pradesh","District":"Hyderabad","Division":"Hyderabad South East",
+"Region":"Hyderabad City","Block":"Saroornagar","State":"Telangana","Country":"India","Pincode":"500060"}]}]
+
+b. GET https://api.postalpincode.in/postoffice/{POSTOFFICEBRANCHNAME}   -- which gives number of post offices on
+specific given city name and it's sub post offices complete addresses.
+
+[{"Message":"Number of Post office(s) found:2","Status":"Success","PostOffice":[{"Name":"New Delhi ",
+"Description":null,"BranchType":"Head Post Office","DeliveryStatus":"Delivery","Circle":"Delhi",
+"District":"New Delhi","Division":"New Delhi GPO","Region":"Delhi","State":"Delhi","Country":"India",
+"Pincode":"110001"},{"Name":"New Delhi South Ext-II","Description":null,"BranchType":"Sub Post Office",
+"DeliveryStatus":"Non-Delivery","Circle":"Delhi","District":"South Delhi","Division":"New Delhi South",
+"Region":"Delhi","State":"Delhi","Country":"India","Pincode":"110049"}]}]
+
+
+
+
+
+3. // Signup and First call to the Weather API. Response data from the API
```

Comment:
It looks like you didn't finish this section, but the above explanation is actually quite good already, I'd say 🙂

Reply:
Hi Thomas, thank you for your feedback. The third one in the file "apiresponse.js" is actually the json response after signed and called the weather API. Plz check it once.